### PR TITLE
update reorg

### PR DESCRIPTION
**Pull request by siddontang**

### CODE REVIEW

**Comment by siddontang** [comment]
@qiuyesuifeng

File: ddl/reorg.go [hunk 1]
```diff
@@ -108,28 +108,54 @@ func (d *ddl) runReorgJob(f func() error) error {
...
 
+func (d *ddl) isOwnerInReorg(txn kv.Transaction) error {
+	t := meta.NewMeta(txn)
+	owner, err := t.GetDDLOwner()
+	if err != nil {
+		return errors.Trace(err)
+	} else if owner == nil || owner.OwnerID != d.uuid {
+		// if no owner, we will try later, so here just return error.
+		// or another server is owner, return error too.
+		return errors.Trace(ErrNotOwner)
```
Comment (qiuyesuifeng) [inline ddl/reorg.go]: Any need to add a log for tracing?

Comment (siddontang) [inline ddl/reorg.go]: no, outer will log this error.

**Comment by qiuyesuifeng** [comment]
LGTM

**Pull request merged**